backend/python/app/services/implementations/story_service.py

```diff
@@ -424,6 +424,25 @@ def update_story_translation_content_status(
             status,
         )
 
+    def resolve_all_story_translation_content(self, story_translation_id):
```

Comment:
nit: we should check that the story stage is in review. My bad, I shouldve caught this earlier - we'll need it for `update_story_translation_content_status` too. services like `create_comment` have examples